### PR TITLE
google_analytics_4 fix

### PR DESCRIPTION
**Pull request by jiwalker-usgs**
Bunch of weird newline changes, but main change is to add slow_fetch and check if result has any data

### CODE REVIEW

File: scripts/fetch/fetchGA.R [hunk 1]
```diff
@@ -6,60 +6,61 @@ library(data.table)
...
       needToUpdate <- bind_rows(needToUpdate, newIDs)
-      
+
       if(nrow(needToUpdate) > 0) {
         message("Sciencebase file is out of date, updating from GA")
         new_GA_DF <- data.frame()
         gar_auth_service(file.path(Sys.getenv("HOME"), ".vizlab/VIZLAB-a48f4107248c.json"))
-        for(i in 1:nrow(needToUpdate)) { 
-          #NOTE: only want to pull full days, so don't pull today's data!  
+        for(i in 1:nrow(needToUpdate)) {
```
Comment (ldecicco-USGS) [inline scripts/fetch/fetchGA.R]: Might be worth:
`seq_along(needToUpdate)`

Comment (jiwalker-usgs) [inline scripts/fetch/fetchGA.R]: yeah, that would be good

Comment (jiwalker-usgs) [inline scripts/fetch/fetchGA.R]: actually, that would be for traversing columns

Comment (ldecicco-USGS) [inline scripts/fetch/fetchGA.R]: seq_len(nrow(df))

**Comment by ldecicco-USGS** [comment]
merge? looks good to me

**Comment by jiwalker-usgs** [comment]
yeah, we might still run into the duplicate error though

**Pull request merged**